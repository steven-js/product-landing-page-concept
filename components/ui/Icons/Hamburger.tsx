const Hamburger = () => {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="18"
      height="16"
      fill="none"
      viewBox="0 0 18 16"
    >
      <path
        fill="var(--neutral-neutral-1000)"
        d="M0 .5h18V2H0V.5ZM0 14h18v1.5H0V14Zm0-9h18v1.5H0V5Zm0 4.5h18V11H0V9.5Z"
      />
    </svg>
  )
}

export default Hamburger
