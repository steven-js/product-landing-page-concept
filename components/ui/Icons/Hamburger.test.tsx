import { render } from '@testing-library/react'
import Hamburger from './Hamburger'

describe('Hamburger icon test', () => {
  it('should render the Hamburger icon unchanged', () => {
    const { container } = render(<Hamburger />)
    expect(container).toMatchSnapshot()
  })
})
