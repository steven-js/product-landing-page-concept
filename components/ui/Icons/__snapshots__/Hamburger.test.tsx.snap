// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hamburger icon test should render the Hamburger icon unchanged 1`] = `
<div>
  <svg
    fill="none"
    height="16"
    viewBox="0 0 18 16"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 .5h18V2H0V.5ZM0 14h18v1.5H0V14Zm0-9h18v1.5H0V5Zm0 4.5h18V11H0V9.5Z"
      fill="var(--neutral-neutral-1000)"
    />
  </svg>
</div>
`;
